uReplicator-Manager/src/test/java/com/uber/stream/kafka/mirrormaker/manager/rest/TestManagerTopicManagement.java

```diff
@@ -67,7 +72,7 @@ public void shutdown() {
     LOGGER.info("Trying to stop worker");
     for (int i = 0; i < WORKER_STARTER1.size(); i++) {
       WORKER_STARTER1.get(i).shutdown();
-      WORKER_STARTER2.get(i).stop();
+      WORKER_STARTER2.get(i).shutdown();
       WORKER_STARTER3.get(i).disconnect();
```

Comment:
why is this disconnect?

Reply:
I guess the test code here meant to test different ways to shutdown worker, however, the "stop" method is not available in new worker